### PR TITLE
Merge SystemID deck functionality into master

### PR DESCRIPTION
**Pull request by tobbeanton**
Support for the SystemID deck has been in the dev-systemId branch but is constantly getting outdated. With the Kbuild system this can now be an easy configured with "make sysid_defconfig".

### CODE REVIEW

**Comment by whoenig** [comment]
excellent idea

**Review by knmcguire — CHANGES_REQUESTED** [review]
I don't see an objection of just merging this since it is supposed to be behind a build flag, but I'm a bit worried about the DWM1000 library submodule.... why was this updated ? Are we sure this will not cause issues with our LPS implementation?

File: tools/system_id/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+* CF Mount (3D printed)
+
+Mount CF upside down (to avoid downwash)
+
+## Measurement
+
+### Common
+
+1. Mount CF
+2. Switch to crazyflie-lib-python with branch `feature-emergency-stop` in order to be able to CTRL+C a script safely.
```
Comment (gemenerik) [inline tools/system_id/README.md:26]: Is this still required?

File: tools/system_id/plot_motor_delay.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
```
[the remark this answers is not in the document]
Comment (gemenerik) [reply tools/system_id/plot_motor_delay.py:2]: Axes3D

File: tools/system_id/README.md (hunk 1 above)
Comment (tobbeanton) [reply tools/system_id/README.md:26]: It wasn't required before either but can be handy so I'm going to leave it like that.

File: tools/system_id/plot_data.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    plt.show()
+
+    fig, ax = plt.subplots()
+
+    print(data[0, 7]/vbat[0])
+
+    # ax.plot(thrust, vbat, label='pm.vbat')
+    # ax.plot(thrust, data[:, 7]/1.0955852742562724, label='i2c.v_avg')
+    # ax.plot(rpm, data[:, 8], label='i2c.i_avg')
```
[the remark this answers is not in the document]
Comment (tobbeanton) [reply tools/system_id/plot_data.py:45]: Most of these scrips are experimental and I think the commented-out can help so I'm going to leave it there.

File: tools/system_id/plot_motor_delay.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+c20 = 20.573302305476638
+c11 = 38.42885066644033
+
+def pwm2force(pwm, v):
+	pwm = pwm / 65536
+	v = v / 4.2
+	return c00 + c10 * pwm + c01 * v + c20 * pwm * pwm + c11 * pwm * v
+
+
+# u dot = lambda (u_des - u)
```
[the remark this answers is not in the document]
Comment (tobbeanton) [reply tools/system_id/plot_motor_delay.py:25]: It's experimental code so going to leave it there

File: tools/system_id/plot_motor_delay.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+# # compute f desired from pwm
+# f_des = []
+# for i in range(0, data.shape[0]):
+# 	f_des.append(pwm2force(data[i,2], data[i,3]) * 4)
+# f_des = np.array(f_des)
+
+# ax1.plot((data[:,0] - data[0,0])/1e3, f_des, 'g', label="Desired")
+# ax1.plot((data[:,0] - data[0,0])/1e3, u_pred, 'r', label="Prediction")
+# ax1.plot((data[:, 0] - data[0, 0])/1e3, data[:, 2], 'r', label="PWM")
```
[the remark this answers is not in the document]
Comment (tobbeanton) [reply tools/system_id/plot_motor_delay.py:59]: It's experimental code so going to leave it there

**Pull request merged**